python/lsst/faro/base/MatchedCatalogBase.py

```diff
@@ -227,6 +233,7 @@ def get_box_wcs(self, skymap, oid):
 
     def runQuantum(self, butlerQC, inputRefs, outputRefs):
         inputs = butlerQC.get(inputRefs)
+        self.log.verbose("Inputs obtained from the butler.")
```

Comment:
FYI I added a periodic logger to `butlerQC.get` so there will be a message issued if it takes a long time. It does no harm to have an extra message here with the task logger.